packages/ignite-basic-structure/index.js

```diff
@@ -54,6 +54,9 @@ const add = async function (context) {
     filesystem.copy(`${sourceFolder}/App`, `${process.cwd()}/App`, { overwrite: true })
   }
 
+  // Create ./ignite/plugins
+  filesystem.dir(`${process.cwd()}/ignite/plugins`)
```

Comment:
We'd have to change `add` to not use the node-modules folder.  Maybe I'm not understanding.  This feels like it might get complex pretty fast.